web/react/pages/channel.jsx

```diff
@@ -70,8 +69,8 @@ function setupChannelPage(props) {
     // Modals
     //
     ReactDOM.render(
-        <GetLinkModal />,
-        document.getElementById('get_link_modal')
+        <GetTeamInviteLinkModal />,
```

Comment:
This shouldn't render here. We should have a separate place for "Global" modals. No need to fix this now. Just a reminder that this should be fixed when I move everything out of here.